include/mruby.h

```diff
@@ -943,11 +954,11 @@ typedef struct mrb_kwargs mrb_kwargs;
 
 struct mrb_kwargs
 {
-  uint32_t num;
-  mrb_value *values;
-  const char *const *table;
-  uint32_t required;
-  mrb_value *rest;
+  uint32_t num;                 /* number of keyword arguments */
+  uint32_t required;            /* number of required keyword arguments */
+  mrb_sym *table;               /* C array of symbols for keyword names */
```

Comment:
I think it's better to define it as `const mrb_sym *table` so that you can pass an array of `const mrb_sym`.

Reply:
Indeed. Thank you.